components/tools/OmeroPy/src/omero/plugins/import.py

```diff
@@ -320,6 +499,140 @@ def importer(self, args):
             if err:
                 err.close()
 
+    def bulk_import(self, command_args, xargs):
+
+        try:
+            from yaml import safe_load
+        except ImportError:
+            self.ctx.die(105, "yaml is unsupported")
+
+        old_pwd = os.getcwd()
+        try:
+
+            # Walk the .yml graph looking for includes
+            # and load them all so that the top parent
+            # values can be overwritten.
+            contents = list()
+            bulkfile = command_args.bulk
+            while bulkfile:
+                bulkfile = os.path.abspath(bulkfile)
+                parent = os.path.dirname(bulkfile)
+                with open(bulkfile, "r") as f:
+                    data = safe_load(f)
+                    contents.append((bulkfile, parent, data))
+                    bulkfile = data.get("include")
+                    os.chdir(parent)
+                    # TODO: include file are updated based on the including file
+                    # but other file paths aren't!
```

Comment:
No. All the cleanup for these metadata(52) PRs typically happens when opened against the mainline.